examples/canjs/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+!node_modules/canjs-localstorage/can.localstorage.js
+!node_modules/canjs/can.jquery.js
+!node_modules/jquery/dist/jquery.js
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-app-css/index.css
+!node_modules/todomvc-common/base.css
```

Comment:
You can't really do this. If you ignore a folder, you can't unignore anything inside it. See the other upgrades on how to do it.